### PR TITLE
Tpetra:  more ascicgpu tests

### PR DESCRIPTION
**Pull request by kddevin**
@trilinos/tpetra 

## Motivation

Adding tests for tpetraCrsRefactor branch #8930 akin to those for TpetraDualViewRefactor branch #8821 

## Testing

NA:  these files add more tests

### CODE REVIEW

File: cmake/ctest/drivers/ascicgpu031/ctest_linux_nightly_mpi_debug_nouvm_ascicgpu031_crs.cmake [hunk 1]
```diff
@@ -86,6 +86,7 @@ SET(EXTRA_CONFIGURE_OPTIONS
 
   "-D Kokkos_ENABLE_CUDA_UVM=OFF"
 
+  "-D Tpetra_ENABLE_DEPRECATE_CODE=OFF"
```
Comment (tasmith4) [inline cmake/ctest/drivers/ascicgpu031/ctest_linux_nightly_mpi_debug_nouvm_ascicgpu031_crs.cmake:89]: maybe a typo here?  should be DEPRECATED I think

File: cmake/ctest/drivers/ascicgpu031/CMakeLists.txt [hunk 2]
```diff
@@ -27,11 +27,26 @@ TRILINOS_DRIVER_ADD_DASHBOARD(
 
 TRILINOS_DRIVER_ADD_DASHBOARD(
   DEBUG_CUDA_NO_UVM_CRS
-  ctest_linux_nightly_mpi_release_nouvm_crs_ascicgpu031.cmake
+  ctest_linux_nightly_mpi_debug_nouvm_crs_ascicgpu031.cmake
```
Comment (tasmith4) [inline cmake/ctest/drivers/ascicgpu031/CMakeLists.txt:30]: seems like the "crs" is in front of "ascicgpu031" in this file, but afterwards in the actual filename

**Review by tasmith4 — CHANGES_REQUESTED** [review]

**Comment by kddevin** [comment]
Good catches, @tasmith4 .  I just pushed fixes.

**Review by GeoffDanielson — APPROVED** [review]

**Comment by prwolfe** [comment]
This PR will not merge until the requested change conversation is marked as resolved. It looks like this is stopping the autotester from running other jobs at the moment (not sure why - see #8981.) @allevin - do you know why this would cause an unrecoverable exception in the sutotester?

**Review by tasmith4 — APPROVED** [review]

**Comment by tasmith4** [comment]
@prwolfe I approved and merged so hopefully the autotester can recover now.

**Comment by allevin** [comment]
> This PR will not merge until the requested change conversation is marked as resolved. It looks like this is stopping the autotester from running other jobs at the moment (not sure why - see #8981.) @allevin - do you know why this would cause an unrecoverable exception in the sutotester?

@prwolfe @tasmith4 - This is a bit of the AT and Github combined causing the issue.  The PR was marked as Mergeable by Github.  And there was a valid approved review for the last commit.   The problem is that there was a “requesting change” review on a previous commit.  For some reason, since the “requesting change” was not cleared, Github deemed the PR as not able to be merged (even though it was marked as mergeable).  The conflicting info was causing the AT to fail and prevented further processing of later PR's.   I will be investigating possible changes to the AT to catch this condition.

**Comment by allevin** [comment]
> This PR will not merge until the requested change conversation is marked as resolved. It looks like this is stopping the autotester from running other jobs at the moment (not sure why - see #8981.) @allevin - do you know why this would cause an unrecoverable exception in the sutotester?

@prwolfe @tasmith4 - This is a bit of the AT and Github combined causing the issue.  The PR is marked as Mergeable by Github.  And there was a valid approved review for the last commit.   The problem is that there was a “requesting change” review on a previous commit.  For some reason, since the “requesting change” was not cleared, Github deemed the PR as not able to be merged.

> This PR will not merge until the requested change conversation is marked as resolved. It looks like this is stopping the autotester from running other jobs at the moment (not sure why - see #8981.) @allevin - do you know why this would cause an unrecoverable exception in the sutotester?

@prwolfe You are correct, the failure on #8977 was blocking further processing of this PR.

**Pull request merged**